conf/prod.routes

```diff
@@ -9,6 +9,7 @@ POST        /registration/02.00.00/individual                uk.gov.hmrc.cgtprop
 POST        /registration/:entityType/:idType/:idValue       uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.BusinessPartnerRecordController.getBusinessPartnerRecord(entityType, idType, idValue)
 
 POST        /subscriptions/create/CGT                          uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.SubscriptionController.subscribe()
+GET         /subscriptions                                     uk.gov.hmrc.cgtpropertydisposalsstubs.controllers.SubscriptionController.getSubscriptionDetails(regime : String, id : String)
```

Comment:
the URL in the DES spec doesn't have query parameters - the form of the url is `/subscriptions/{regime}/{idType}/{idValue}`

Reply:
I see this in the spec: /subscriptions?regime={regimeValue}&id={idValue}